### PR TITLE
compat: Support Bokeh 3.6

### PR DESCRIPTION
**Pull request by hoxbro**
With the release of Panel 1.5.1 this should now be possible.

### CODE REVIEW

**Comment by hoxbro** [comment]
Maybe failing install is related to https://github.com/bokeh/bokeh/pull/14085.

**Comment by mattpap** [comment]
I fixed type errors in edit tools. I'm still investigating what to do about these tools in general. There are more differences between geoviews' and bokehjs' implementation that I originally expected.

File: pyproject.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling", "hatch-vcs", 'bokeh ==3.5']
+requires = ["hatchling", "hatch-vcs", 'bokeh >=3.5.0,<3.7.0']
```
Comment (hoxbro) [inline pyproject.toml:2]:
```suggestion
requires = ["hatchling", "hatch-vcs", 'bokeh ==3.6.0']
```

File: pixi.toml [hunk 2]
```diff
@@ -23,7 +23,7 @@ lint = ["py311", "lint"]
 download-data = ["download-data"]
 
 [dependencies]
-bokeh = "3.5.*"
+bokeh = ">=3.5.0,<3.7.0"
```
Comment (hoxbro) [inline pixi.toml:26]:
```suggestion
bokeh = "3.6.*"
```

File: pyproject.toml [hunk 3]
```diff
@@ -27,7 +27,7 @@ classifiers = [
 ]
 
 dependencies = [
-    'bokeh >=3.5.0,<3.6.0',
+    'bokeh >=3.5.0,<3.7.0',
```
Comment (hoxbro) [inline pyproject.toml:30]:
```suggestion
    'bokeh >=3.6.0,<3.7.0',
```

File: pyproject.toml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["hatchling", "hatch-vcs", 'bokeh ==3.5']
+requires = ["hatchling", "hatch-vcs", 'bokeh ==3.6.0']
```
Comment (hoxbro) [inline pyproject.toml:2]:
```suggestion
requires = ["hatchling", "hatch-vcs", 'bokeh ==3.6']
```

**Pull request merged**